### ISSUE TITLE
Visual label name of "Jobs" and "Endpoints" controls is not there in programmatic name: A11y_Azure Machine Learning Extension for VS Code_Tree View_Label in name

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode-tools-for-ai#2487]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_Azure Machine Learning Extension for VS Code_Win32_Jan2024; #Azure Machine Learning Extension for VS Code; #WCAG2.5.3; #FTP; #Label In name; #A11yWCAG2.1;

## Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.0
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch the VS Code.
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
4. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
5. The Azure panel will now be active next to the toolbar.
6. Tab until the Machine Learning dropdown.
7. Tab till any subscription for example AML V1 Personal subscription and press enter to expand tree view.
8. Tab till any workspace and press ENTER to expand tree view
9. TAB till "Jobs" and "Endpoints" and inspect it
10. Observer that visual name of "Jobs" and "Endpoints" is not there in programmatic name.

## Actual Experience
Visual label name of "Jobs" and "Endpoints" controls is not there in programmatic name.

## Expected Experience
Visual label name of "Jobs" and "Endpoints" controls should be in the start of programmatic name.

## User Impact:
Speech input user will face difficult to interact with control because visual label name of "Jobs" and "Endpoints" is not there in programmatic name.

## Attachment
![Image](https://github.com/user-attachments/assets/4aec9220-ccef-4374-b8e7-29b970a9c9ae)

![Image](https://github.com/user-attachments/assets/503e6870-d2a0-4626-ab77-b0563bccc16a)

**Comment by JerryYangKai** [comment]
Hello @kapilvaishna 
It has been fixed, please install the latest pre-release version to verify.

![Image](https://github.com/user-attachments/assets/57d14e39-670e-4201-8af9-6ddc0f457585)
